packages/fx-core/src/plugins/resource/bot/languageStrategy.ts

```diff
@@ -29,7 +30,9 @@ export class LanguageStrategy {
       {
         group: group_name,
         lang: utils.convertToLangKey(config.scaffold.programmingLanguage!),
-        scenario: TemplateProjectsConstants.DEFAULT_SCENARIO_NAME,
+        scenario: config.actRoles.includes(PluginActRoles.Notification)
```

Comment:
Since we may have another scenario of "Command and response", can we encapsulate them as scenario concept (consider default as another scenario) and have a method to resolve it (instead of simply if else)? It would be better to maintain and be extensible to support future scenarios.

Reply:
It makes sense. Add a helper function to resolve bot scenario